.storybook/addons.js

```diff
@@ -0,0 +1,2 @@
+import '@storybook/addon-options/register';
```

Comment:
NIT: Are these linted? Seeing different quotation marks than the rest of the stuff.